### PR TITLE
Add AppSec Kit documentation for Vaadin 7

### PR DESCRIPTION
**Pull request by tamasmak**
Closes [#41](https://github.com/vaadin/appsec-kit/issues/41)

### CODE REVIEW

File: articles/appseckit/getting-started.asciidoc [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+----
+mvn jetty:run
+----
+
+
+=== Open AppSec Kit UI
+
+==== Notification Dialog
+
+After the application has started, AppSec Kit analyzes the dependencies, collects the vulnerabilities if any exists, and displays a `Notification Dialog` (Figure 1.) where you can open the `UI` in a new tab.
```
Comment (jouni) [inline articles/appseckit/getting-started.asciidoc:89]:
```suggestion
After the application has started, AppSec Kit analyzes the dependencies, collects the vulnerabilities if any exists, and displays the <<getting-started-notification-dialog>> where you can open the `UI` in a new tab.
```

**Review by jouni — COMMENTED** [review]
Use cross references to point to the images/figures, to avoid duplicating the title and using explicit numbering which might go out of sync.

File: articles/appseckit/getting-started.asciidoc (hunk 1 above)
Comment (tamasmak) [reply articles/appseckit/getting-started.asciidoc]: Good point! Changed to use cross-references everywhere and in the V8 docs too.

**Comment by heruan** [comment]
If language check is not happening today, can we merge this and do the check later?

**Review by heruan — APPROVED** [review]

**Comment by jouni** [comment]
What's the hurry? 🤗

**Pull request merged**